docs/docs/10-users/60-sessions/guides/45-use-your-own-docker-image-for-renku-session.md

```diff
@@ -34,6 +34,10 @@ In the project page:
             - `continuumio/anaconda3:2024.06-1`
         - if the image is hosted on gitlab.renkulab.io:
             - `registry.renkulab.io/laura.kinkead1/n2o-pathway-analysis:980f4a3`
+		- if you want to use a custom image built with Github Actions:
+    		- `ghcr.io/<username>/<projectname>:sha-<hash>`
+			- `ghcr.io/swissdatasciencecenter/renku-custom-vscodium-image-template:sha-a6e42d6`
```

Comment:
this is only the case for the actual template - it's not something the user would use

```suggestion
```